rewrite/to_ir.rs

```diff
@@ -119,109 +41,145 @@ impl<'i> ParseTreeVisitor<'i, CypherParserContextType> for FilterVisitor {
 }
 
 impl<'i> CypherVisitor<'i> for FilterVisitor {
-    /// This function visits a match clause.  It extracts the graph inside, complete with any
-    /// node attributes, and stores that information in a struct_filter.  It then extracts any information
-    /// in the where clause, which pertains to the whole graph, and stores that in an attribute_filter.
-    fn visit_oC_Match(&mut self, ctx: &OC_MatchContext<'i>) {
+    // we do not want to visit returns in this case, ignore that part of the tree
+    // TODO: Apply the visitor directly to the MATCH body instead
+    fn visit_oC_Return(&mut self, _ctx: &OC_ReturnContext<'i>) {}
+
+    fn visit_oC_PropertyOrLabelsExpression(
+        &mut self,
+        ctx: &OC_PropertyOrLabelsExpressionContext<'i>,
+    ) {
+        log::info!("Generating IrReturn");
```

Comment:
I think this logging is off.  Generating the IrReturn happens in the return visitor, right?

Reply:
Logging is set in main. It should work. `IrReturn` is a general purpose element that is filled in OC_PropertyOrLabelsExpressionContext. It is used in both visitors. Maybe we should find a better name for it.